src/terminal/adapter/ut_adapter/adapterTest.cpp

```diff
@@ -1824,17 +1824,17 @@ class AdapterTest
 
         Log::Comment(L"Test 1: Verify response for unsolicited permission.");
         _testGetSet->PrepData();
-        VERIFY_IS_TRUE(_pDispatch->RequestTerminalParameters(DispatchTypes::ReportingPermission::Unsolicited));
+        _pDispatch->RequestTerminalParameters(DispatchTypes::ReportingPermission::Unsolicited);
         _testGetSet->ValidateInputEvent(L"\x1b[2;1;1;128;128;1;0x");
 
         Log::Comment(L"Test 2: Verify response for solicited permission.");
         _testGetSet->PrepData();
-        VERIFY_IS_TRUE(_pDispatch->RequestTerminalParameters(DispatchTypes::ReportingPermission::Solicited));
+        _pDispatch->RequestTerminalParameters(DispatchTypes::ReportingPermission::Solicited);
         _testGetSet->ValidateInputEvent(L"\x1b[3;1;1;128;128;1;0x");
 
         Log::Comment(L"Test 3: Verify failure with invalid parameter.");
         _testGetSet->PrepData();
-        VERIFY_IS_FALSE(_pDispatch->RequestTerminalParameters((DispatchTypes::ReportingPermission)2));
+        _pDispatch->RequestTerminalParameters((DispatchTypes::ReportingPermission)2);

```

Comment:
This test has a `RequestTerminalParameters` at the end. Can it still be removed?

Reply:
If you mean the "Test 4" below this - that one can still be kept since it's testing the API throw being propagated. I'm not sure it has a whole lot of value, but it's at least doing something. You may want to renumber it though if we're dropping test 3.